src/state/actions/dispatchArIOInteraction.ts

```diff
@@ -42,6 +44,7 @@ export default async function dispatchArIOInteraction({
   signer?: ContractSigner;
   ao?: AoClient;
   scheduler?: string;
+  fundFrom?: AoGetCostDetailsParams['fundFrom'];
```

Comment:
Why not use use FundFrom type here instead of indexed type?

Reply:
I didnt know that was an export :P will change